test/jasmine/tests/config_test.js

```diff
@@ -551,230 +551,232 @@ describe('config argument', function() {
         });
     });
 
-    describe('responsive figure', function() {
-        var gd;
-        var data = [{type: 'scatter', x: [1, 2, 3, 4], y: [5, 10, 2, 8]}];
-        var width = 960;
-        var height = 800;
-
-        var parent, elWidth, elHeight;
-
-        beforeEach(function() {
-            viewport.set(width, height);
-
-            // Prepare a parent container that fills the viewport
-            parent = document.createElement('div');
-            parent.style.width = '100vw';
-            parent.style.height = '100vh';
-            parent.style.position = 'fixed';
-            parent.style.top = '0';
-            parent.style.left = '0';
-        });
-
-        afterEach(function() {
-            Plotly.purge(gd); // Needed to remove all event listeners
-            document.body.removeChild(parent);
-            viewport.reset();
-        });
+    ['scatter', 'scattergl'].forEach(function(traceType) {
```

Comment:
> We should add the traceType value inside these its so that if the scatter or the scattergl cases fail, we'll know which one did. For example,

I think this line allows to clearly identify which case fails. Am I mistaken?

cc @etpinard

Reply:
Running `npm run test-jasmine -- config` will a failing test gives me:
![screenshot_2019-01-25_12-35-18](https://user-images.githubusercontent.com/301546/51762435-b7f8c900-209d-11e9-8691-43b9bf60c793.png)

What am I missing?

Thank you @etpinard

Reply:
`scatter` or `scattergl`.

Reply:
Maybe it's not prominent enough but it's already there:
![screenshot_2019-01-25_12-35-18](https://user-images.githubusercontent.com/301546/51763481-b8469380-20a0-11e9-9984-4324da180dfa.png)

Reply:
Oops. It's in the `describe` block:

```
describe('responsive ' + traceType + ' trace', function() {
```

My bad.